BuildTools/.swiftlint-source.yml

```diff
@@ -84,9 +84,10 @@ excluded:
   - ${SRCROOT}/bundle
   - ${SRCROOT}/scripts/genstrings.swift
   - ${SRCROOT}/danger/DangerTests.swift
-  - ${SRCROOT}/*Tests
   - ${SRCROOT}/*UITests
   - ${SRCROOT}/Tests
+  - ${SRCROOT}/*Tests
```

Comment:
Not really sure if this was a type and should've been `${SRCROOT}/*/Tests`, but I decided to keep it around just in case it was there for a reason.